frontend/src/services/http/http.service.ts

```diff
@@ -20,7 +20,7 @@ class Http {
       .catch(this._throwError);
   }
 
-  _getHeaders(contentType?: ContentType): Headers {
+  private _getHeaders(contentType?: ContentType): Headers {
```

Comment:
I think it would be better to omit the underscore or replace it with #, as is already done with private properties

Reply:
Good catch 👍